src/theme/themes/plh_kids_kw/_overrides.scss

```diff
@@ -346,51 +345,45 @@ body[data-theme="plh_kids_kw"] {
       margin-bottom: 20px;
 
       &[data-variant~="gray"] {
-        --border-color: none;
-        --background-color: none;
         .text-bubble {
           border: 1px solid var(--color-outline);
           background-color: var(--color-white-variant);
         }
       }
 
       &[data-variant~="primary"] {
-        --background-color: none;
-        --border-color: none;
         .text-bubble {
-          border: 1px solid var(--ion-color-secondary-700);
-          background-color: var(--ion-color-primary-50);
+          border: 1px solid var(--ion-color-secondary-700) !important;
+          background-color: var(--ion-color-primary-50) !important;
         }
       }
 
       &[data-variant~="secondary"] {
-        --background-color: none;
-        --border-color: none;
         .text-bubble {
-          border: 1px solid var(--color-accent-yellow-40);
-          background-color: var(--color-white-variant);
+          border: 1px solid var(--color-accent-yellow-40) !important;
+          background-color: var(--color-surface-white-variant) !important;
         }
       }
 
-      &[data-variant~="accent-1"] {
+      &[data-variant~="speaker-3"] {
         margin-bottom: 14px;
         .text-bubble {
-          border: 1px solid var(--color-accent-orange-70);
-          background-color: var(--color-accent-orange-95);
+          border: 1px solid var(--color-accent-orange-70) !important;
+          background-color: var(--color-accent-orange-95) !important;
         }
       }
 
-      &[data-variant~="accent-2"] {
+      &[data-variant~="speaker-4"] {
         margin-bottom: 14px;
         .text-bubble {
-          border: 1px solid var(--color-accent-yellow-70);
-          background-color: var(--color-accent-yellow-95);
+          border: 1px solid var(--color-accent-yellow-70) !important;
```

Comment:
nit(blocking): as described in comment on component style file, these overrides are better applied by overriding the exposed component style variables in the theme's index file